core/services/job/models.go

```diff
@@ -823,6 +823,79 @@ func (s *GatewaySpec) SetID(value string) error {
 	return nil
 }
 
+// NodeServerConfigPath returns NodeServerConfig.Path or empty string, if not found or it's not a string
+func (s *GatewaySpec) NodeServerConfigPath() string {
+	if nsc, ok := s.GatewayConfig["NodeServerConfig"]; ok {
```

Comment:
@Tofel I'd recommend just unmarshaling the config into a struct (I would expect this to already exist somewhere) and then just grabbing what you need from it; same with the function below.

Reply:
It doesn't!

Reply:
well, can't use due to circular imports :/ and I don't want to put that and other types in yet another package to resolve it, I think it should be done in another PR